### ISSUE TITLE
Misleading rendering BiDi text in parse-integer example

### ISSUE THREAD

**Issue by kosek** [qt4cg/qtspecs#1113]
In fuction index one example of fn:parse-integer() is using parameters containing arabic letters. This leads to wrong display of parameters as 1st and 2nd parameter look like switched because browser renders both of them from left to right. It is this example:
````
<fos:test>
               <fos:expression><eg>translate('٢٠٢٣', '٠١٢٣٤٥٦٧٨٩', '0123456789')
=> parse-integer()</eg></fos:expression>
               <fos:result>2023</fos:result>
            </fos:test>
````
This looks confusing. I don't know what will be the best fix. Maybe storing '٠١٢٣٤٥٦٧٨٩' into variable would help and prevent the issue.

**Comment by michaelhkay** [comment]
FYI, the example is displaying correctly for me in Safari on Mac, and incorrectly in Chrome on Mac. Where "correctly" means that the first (4-character) argument is shown first.

I think I encountered the same problem with the sentence (also in format-integer) 

> A format token of &#x661; (Arabic-Indic digit one) generates the sequence ١ then ٢ then ٣ ...

where I had to use the word "then" rather than commas between the values to ensure correct rendering.
